packages/server/modules/shared/middleware/security.ts

```diff
@@ -0,0 +1,8 @@
+import type express from 'express'
+
+export const crossOriginResourcePolicyMiddleware =
+  (value: 'same-origin' | 'cross-origin' | 'same-site'): express.RequestHandler =>
+  (req, res, next) => {
+    res.setHeader('Cross-Origin-Resource-Policy', value)
```

Comment:
any cross-origin resources (previews, attachments etc.) coming from server must have this header set or the FE will be blocked from receiving them